apps/account/templates/account/detail.html

```
@@ -1,217 +1,217 @@
+                    <p class="text-uppercase small text-muted mb-1">{% trans "Push notifications" %}</p>
+                    <p class="fw-semibold mb-0">
+                      {% if user.wants_to_receive_webpush_notifications %}
+                        {% trans "Enabled" %}
+                      {% else %}
+                        {% trans "Disabled" %}
+                      {% endif %}
+                    </p>
+                    <p class="text-muted small mb-0">{% trans "Manage this toggle from the edit screen." %}</p>
+                  </div>
+                  <div class="form-check form-switch m-0">
+                    <input class="form-check-input"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)